North-Seattle-College/ad440-winter2022-thursday-repo
# nlp script to run in Sagemaker

Updated NLP script from the frontend team made to process Floop comment data (with Spacy tokenization) in Jupiter Notebook format
To test:
open nltk-json.ipynb file in Jupyter Notebook or VSCode (you might need to install a plugin to work with these kinds of files)
Run the first cell which will install the Python dependencies to run the code
run the second cell which should process the json file and produce the result and the graph like this:
![Screenshot 2022-02-10 181900](https://user-images.githubusercontent.com/17448320/153528189-01b5c48d-fa1e-49f4-b4f1-5d7dc82366e3.png)
